client/components/ErrandTracker/ErrandTrackerContainer.jsx

```diff
@@ -20,7 +20,7 @@ import errandState from '../../state/atoms/errands';
 
 console.log(GOOGLE_MAPS_API_KEY)
 import TopBar from './TopBar/TopBar.jsx';
-import ErrandMap from './errandMap.jsx';
```

Comment:
I noticed that if this is made UpperCase, my app crashes. I always have to fix it back to camel case (see Louisa's update in Slack).

Reply:
I'll approve this all, but just figured I'd flag it for others' attention

Reply:
I know the filename is `errandMap` so I'd think the import file would have to be camelCase as well